builtin/providers/datadog/resource_datadog_timeboard.go

```diff
@@ -399,40 +397,40 @@ func buildGraphs(terraformGraphs *[]interface{}) *[]datadog.Graph {
 	datadogGraphs := make([]datadog.Graph, len(*terraformGraphs))
 	for i, t_ := range *terraformGraphs {
 		t := t_.(map[string]interface{})
-		datadogGraphs[i] = datadog.Graph{Title: t["title"].(string)}
+		datadogGraphs[i] = datadog.Graph{Title: datadog.String(t["title"].(string))}
 		d := &datadogGraphs[i]
-		d.Definition.Viz = t["viz"].(string)
+		d.Definition.Viz = datadog.String(t["viz"].(string))
 
 		if yaxis_, ok := t["yaxis"]; ok {
 			yaxis := yaxis_.(map[string]interface{})
-			if min_, ok := yaxis["min"]; ok {
-				min, _ := strconv.ParseFloat(min_.(string), 64)
+			if v, ok := yaxis["min"]; ok {
+				min, _ := strconv.ParseFloat(v.(string), 64)
 				d.Definition.Yaxis.Min = &min
 			}
-			if max_, ok := yaxis["max"]; ok {
-				max, _ := strconv.ParseFloat(max_.(string), 64)
+			if v, ok := yaxis["max"]; ok {
+				max, _ := strconv.ParseFloat(v.(string), 64)
 				d.Definition.Yaxis.Max = &max
 			}
-			if scale_, ok := yaxis["scale"]; ok {
-				scale := scale_.(string)
+			if v, ok := yaxis["scale"]; ok {
+				scale := v.(string)
 				d.Definition.Yaxis.Scale = &scale
 			}
 		}
 
-		if autoscale, ok := t["autoscale"]; ok {
-			d.Definition.Autoscale = autoscale.(bool)
+		if v, ok := t["autoscale"]; ok {
+			d.Definition.Autoscale = datadog.Bool(v.(bool))
```

Comment:
Don't think variables that live no longer than 2 lines are worth it. Apart from that I find this easier to parse.